gym/wrappers/test_record_video.py

```diff
@@ -11,10 +11,11 @@
 
 
 def test_record_video_using_default_trigger():
+
     env = gym.make("CartPole-v1")
     env = gym.wrappers.RecordVideo(env, "videos")
     env.reset()
-    for _ in range(199):
```

Comment:
Is this needed?

Reply:
No , I was just debugging  and forgot , I'll revert it